### DISCUSSION TITLE
How to solve the problem of penetrating the model?

### DISCUSSION THREAD

**Discussion by 1783883121** [pixiv/three-vrm#1361] [Help]
Does anyone know how to solve the problem of model penetration?
![image](https://github.com/pixiv/three-vrm/assets/52774084/c2748c15-c1d5-4125-8ad6-ae1d36d44a5d)

**Comment by 0b5vr** [comment]
This is a problem common to any VRM models and implementations, not limited to three-vrm.

**Reply by 1783883121** [reply]
Yes, so is there any way to optimize it?

**Reply by YulesRules** [reply]
@1783883121 not without changing the animation (which would be best practice in this situation) or, if you need it to be able to cope with more dynamic approaches, then adding more colliders and getting overly complicated interactions going on.
